### ISSUE TITLE
NTR: Cytokine storm

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5812]
**Preferred term label:**
Cytokine storm

**Parent term (use hpo.jax.org/app)**
Abnormality of cytokine secretion

see: https://github.com/obophenotype/mammalian-phenotype-ontology/issues/3209

**Comment by pnrobinson** [comment]
Def: Excessive or uncontrolled release of proinflammatory cytokines.
Syn: Hypercytokinemia

Comment: Cytokines are a diverse group of small proteins that are secreted by cells for the purpose of intercellular signaling and communication. Specific cytokines have autocrine, paracrine, and/or endocrine activity and, through receptor binding, can elicit a variety of responses, depending upon the cytokine and the target cell. Among the many functions of cytokines are the control of cell proliferation and differentiation and the regulation of angiogenesis and immune and inflammatory responses. Inflammation associated with a cytokine storm begins at a local site and spreads throughout the body via the systemic circulation. Rubor (redness), tumor (swelling or edema), calor (heat), dolor (pain), and “functio laesa” (loss of function) are the hallmarks of acute inflammation. While the general concept of an excessive or uncontrolled release of proinflammatory cytokines is well known, a precise definition of what constitutes a cytokine storm is lacking. The first use of cytokine storm appears to be in an article published in 1993 on graft-versus-host disease. Cytokine storm has subsequently been described in patients infected by cytomegalovirus, Epstein-Barr virus-associated hemophagocytic lymphohistiocytosis, group A streptococcus, influenza virus, variola virus, severe acute respiratory syndrome coronavirus (SARS-CoV),avian H5N1 influenza virus, and SARS-CoV-2.

**Comment by pnrobinson** [comment]
added as HP_0033041
